dev-libs/dbus-menu-gtk3.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3.6
+# -*- coding: utf-8 -*-
+
+import stdlib
+from stdlib.template import autotools
+from stdlib.manifest import manifest
+from stdlib.template.configure import configure
+
+
+@manifest(
+    name='dbus-menu-gtk3',
+    category='dev-libs',
+    description='''
+    Library to pass a menu structure across DBus.
+    ''',
+    tags=['dbus', 'menu'],
+    maintainer='doom@raven-os.org',
+    licenses=[stdlib.license.License.GPL_V3, stdlib.license.License.LGPL_V2_1, stdlib.license.License.LGPL_V3],
+    upstream_url='https://launchpad.net/libdbusmenu',
+    kind=stdlib.kind.Kind.EFFECTIVE,
+    versions_data=[
+        {
+            'semver': '16.04.0',
+            'fetch': [
+                {
+                    'url': 'https://launchpad.net/libdbusmenu/16.04/16.04.0/+download/libdbusmenu-16.04.0.tar.gz',
+                    'sha256': 'b9cc4a2acd74509435892823607d966d424bd9ad5d0b00938f27240a1bfa878a',
+                },
+                {
+                    'file': 'dbus-menu-gtk3-remove-werror.patch'
```

Comment:
Where's the patch?

Reply:
Somewhere... [over the rainbow](https://www.youtube.com/watch?v=V1bFr2SWP1I).